docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,149 @@
+
+image::images/whats-new-synthetics-monitors.png[]
+
+[discrete]
+=== Enhanced SRE Capabilities
+
+Our synthetic monitoring solution bolsters your SRE experience with dedicated error detail pages and issue triage. Compare successful and failed test runs, analyze performance trends, receive proactive alerts, and integrate with incident management tools for optimized end-user experiences.
+
+image::images/whats-new-synthetics-results.png[]
+
+[discrete]
+=== Streamlined GitOps Management
+
+Our industry-leading synthetic monitoring solution lets you manage monitors as code. Write tests pre-production, then push them into synthetic monitoring for post-production testing. Minimize release-testing lag, reuse scripts, and manage monitors effortlessly. Get version control, RBAC, and centralized test code benefits via source control.
+
+// image::images/whats-new-synthetics-projects.png[]
+
+[discrete]
+=== Broad Coverage Capabilities
+
+Our front-end synthetic monitoring complements APM and Infrastructure Observability, offering critical visibility into your application's workflows and third-party dependencies. Without it, SREs can't fully understand an application's ecosystem, making it vital for optimal performance and exceptional user experiences.
+
+[discrete]
+=== Comprehensive Managed Testing Infrastructure
+
+Our Elastic-first managed service in synthetic monitoring offers global testing coverage with no hardware management needed. Conduct lightweight to full browser tests, and enjoy automatic scaling, built-in security, and seamless Elastic Observability integration. We also support private testing locations for specific needs. Focus on delivering exceptional user experiences with our managed infrastructure.
+
+*Refer to the <<monitor-uptime-synthetics,Synthetic monitoring docs>> for more information.*
+
+
+[discrete]
+== Ingest logs from Amazon Kinesis Data Firehose
+
+https://aws.amazon.com/kinesis/data-firehose/[Amazon Kinesis Data Firehose] is an extract, transform, and load (ETL) streaming data pipeline service that reliably captures, transforms, and delivers streaming data. Users configure data producers to send data to Kinesis Data Firehose and, with {kinesis-firehose-ref}/aws-firehose-setup-guide.html[this new GA integration], Kinesis Data Firehose delivers the data to Elastic Cloud in AWS. Elastic appears as a drop-down option when configuring the Kinesis delivery stream in the AWS console UI, and along with pre-filled default values for most configuration settings, this makes it a direct, easy, agentless setup process to send log data to Elastic. 
+
+image::images/whats-new-amazon-kinesis-data-firehose.png[]
+
+*Refer to the {kinesis-firehose-ref}/aws-firehose-setup-guide.html[setup guide] for more information.*
```

Comment:
@dedemorton this link is broken because there are no `8.8` docs published for the Kinesis docs. Are we planning to publish versioned docs for that book, should we change the [`kinesis-firehose-ref`](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L49) to `https://www.elastic.co/guide/en/kinesis/current`, or is there something else I should do here?

Reply:
@colleenmcginnis Kinesis follows its own versioning strategy (not stack versioning), but there is currently only one version that's in master. I don't understand why the version is resolving to 8.8 when it's set to `master` in https://github.com/elastic/docs/blob/master/shared/versions/stack/8.8.asciidoc

Reply:
I'll open a PR to set it to current for now and see if that works.

Reply:
Oh nevermind. I see what happened. The 8.8 file is not using the correct attribute for the release. I will fix it!

Reply:
@colleenmcginnis OK, so I got confirmation of the versioning: "this is not tied to stack release versions, but for under-the-hood reasons is tied to ESS release milestones. in the future though this we be silently decoupled from the ESS releases." So...stack versions are not appropriate. I will create an attribute like I did for the ESF docs and point it to master for now.

Reply:
Fix is up here: https://github.com/elastic/docs/pull/2688

Reply:
The fix is merged now.